pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -102,13 +105,13 @@ func WithLoadBalancer() InstallOption {
 }
 
 // NewInstaller returns a Tinkerbell StackInstaller which can be used to install or uninstall the Tinkerbell stack
-func NewInstaller(docker Docker, filewriter filewriter.FileWriter, helm Helm, namespace, podCidrRange string) StackInstaller {
+func NewInstaller(docker Docker, filewriter filewriter.FileWriter, helm Helm, namespace string, clusterConfig *v1alpha1.Cluster) StackInstaller {
```

Comment:
Do we really need a complete cluster configuration to achieve stack installation? Adding the cluster object makes it ambiguous as to what is actually needed and how it impacts the behavior of the stack installer.

Reply:
I needed pod cidr config and registry mirror config inside stack installer so I thought of passing a complete cluster configuration thinking we might need more info in future and adding a separate parameters for each values didn't sound right.

Reply:
I would give the installer what it needs and if we find we're continually adding stuff in the future we can adjust then.